scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -1596,12 +1919,49 @@
         # Give some extra gold to the other leaders to make up for the fact that their team is down one leader now:
         [gold]
             side=3 # Bor Cryne
-            amount={ON_DIFFICULTY 20 30 40}
+            amount={ON_DIFFICULTY4 20 30 40 50}
         [/gold]
         [gold]
             side=4 # Uri van Roe
-            amount={ON_DIFFICULTY 10 20 30}
+            amount={ON_DIFFICULTY4 10 20 30 40}
         [/gold]
+        [redraw][/redraw]
+        # ...uh... strange things start to happen after this if you use debug mode to trigger this event before your "allies" have turned on you...
+        # Let's see if this helps:
+        [if]
+            [variable]
+                name=ano_barnon_tr
+                equals=no
+            [/variable]
+            [then]
+                {MSG_Uri _"...well, now that Hoyre's no longer around, there's no need to keep up this farce, is there, Bor?"}
+                [modify_side]
+                    side=4
+                    team_name=bad
+                [/modify_side]
+
+                {REPLACE_SCENARIO_MUSIC theme_of_a_new_order.ogg}
+                {MSG_Cryne _"Indeed, Uri, there isn't! Let our swords instead cut that underling's head off, as well as that witch, his step-mother!"}
+                [modify_side]
+                    side=3
+                    team_name=bad
+                [/modify_side]
+                {APPEND_MUSIC battle/onethousandsuns.ogg}
+                {APPEND_MUSIC battle/battlecry.ogg}
+                {APPEND_MUSIC battle/gameplay06.ogg}
+                {APPEND_MUSIC siege_of_laurelmor.ogg}
+                {APPEND_MUSIC battle-epic.ogg}
+                # This macro (ANO4_TRAITORS) is defined in ano-01_06macros.cfg and contains most of the dialogue that occurs here:
+                {ANO4_TRAITORS _"Reach either eastern or western signpost with both Lorin AND Reme. After that, hold on at least $ano_howmanyturns turns before the honorable death of Gawen Hagarthen."}
```

Comment:
```suggestion
                {ANO4_TRAITORS _"Reach either the eastern or western signpost with both Lorin AND Reme. After that, hold on at least $ano_howmanyturns turns before the honorable death of Gawen Hagarthen."}
```

Reply:
oh I copied and pasted this from somewhere; I'll want to change the original, too...

Reply:
(see lines 603 and 635)

Reply:
ok I changed the originals, too, in [8e3236d](https://github.com/nemaara/A_New_Order/pull/53/commits/8e3236dcb880e37f0d896d4b5b52528d968a6bf8).